File: tacas2020-benchmarks/vaphor/ss2.java
class vapexp{
	static int i,N;
	static long a[];
	static long sum;

static void main()
{
	N=Support.random();

	sum = 0;
	i=0;
	while(i<N)
        {
                a[i] = 1;
		i=i+1;
        }

	i=0;
	while(i<N)
        {
		sum = sum + a[i];
		i=i+1;
        }

	i=0;
	while(i<N)
        {
                a[i] = 0;
		i=i+1;
        }

	i=0;
	while(i<N)
        {
		a[i] = a[i] + sum;
		i=i+1;
        }

	i=0;
	while(i<N)
        {
		sum = sum + a[i];
		i=i+1;
        }

	i=0;
	while(i<N)
        {
		a[i] = a[i] + sum;
		i=i+1;
        }

	i=0;
	while(i<N)
        {
		assert(a[i] == N * (N + 2));
		i=i+1;
        }

}

}

File: tacas2020-benchmarks/booster/ss2.c
#define N

int main()
{
	int i;
	int sum;
	int a[N];

	sum = 0;
	for(i=0; i<N; i++)
	{
		a[i] = 1;
	}

	for(i=0; i<N; i++)
	{
		sum = sum + a[i];
	}

	for(i=0; i<N; i++)
	{
		a[i] = 0;
	}

	for(i=0; i<N; i++)
	{
		a[i] = a[i] + sum;
	}

	for(i=0; i<N; i++)
	{
		sum = sum + a[i];
	}

	for(i=0; i<N; i++)
	{
		a[i] = a[i] + sum;
	}

	for(i=0; i<N; i++)
	{
		assert(a[i] == N*(N+2));
	}
	return 1;
}
